integration_tests/components/component.ts

```diff
@@ -0,0 +1,18 @@
+export type PageElement = Cypress.Chainable<JQuery>
+
+export default abstract class Component {
+  shouldShowKeyAndValue(key: string, value: string): void {
+    cy.get('.govuk-summary-list__key').contains(key).siblings('.govuk-summary-list__value').should('contain', value)
+  }
+
+  shouldShowKeyAndValues(key: string, values: Array<string>): void {
```

Comment:
Did you do something similar to these as `assertDefinition`? I had `Page` inherit `Component` to avoid duplication when pages want to use these

Reply:
Yeah, that was more blind copy/pasting. We don't need this at the moment, so I've got rid